django_project/geosight/data/forms/context_layer.py

```diff
@@ -47,6 +47,26 @@ class ContextLayerForm(forms.ModelForm):
         widget=forms.HiddenInput()
     )
 
+    longitude = forms.CharField(
```

Comment:
Same with model, we could put this as config
config = forms.JsonField(
null=True, blank=True
)

And in frontend we can create separate new inputs and save it to config as json